QuestionsforKaida_s2.md

```diff
@@ -0,0 +1,6 @@
+- How can I refactor the code without repeating the  footer and nav for each page?
+- What is the best way to import google fonts? link or @url import 
```

Comment:
I personally think the nicest way is to download the font file and then import it that way.  (Then your site doesn't depend on Google Fonts always being working, and doesn't send Google tracking information.)